### PR TITLE
CID-15877: Update name for Charac from "Charac Ltd" to "The Pharmacy App" 

### PR DESCRIPTION
**Pull request by Lilian-Chinelo**

### CODE REVIEW

File: src/_data/clients.json [hunk 1]
```diff
@@ -71,6 +70,7 @@
     "remora2": "Remora2",
     "stsft_meditech_patient_portal": "STSFT MEDITECH Patient Portal",
     "tameside_and_glossop_integrated_care_nhs_foundation_trust": "Tameside and Glossop Integrated Care NHS Foundation Trust",
+    "the pharmacy app": "The Pharmacy App",
```
Comment (lps88) [inline src/_data/clients.json:73]: is this spaces where we want underscores?

Comment (ramuned-NHSD) [reply src/_data/clients.json:73]: The ticket only mentions changing client friendly name, so I expected to see  "charac_ltd": "The Pharmacy App"?

File: src/_data/clients.json [hunk 2]
```diff
@@ -11,7 +11,6 @@
     "visit_baby_buddy": "Baby Buddy",
     "boots_uk_ltd": "Boots UK Ltd",
     "visit_chain_sms": "Chain SMS",
-    "charac_ltd": "Charac Ltd",
```
Comment (ramuned-NHSD) [inline src/_data/clients.json:14]: Is zendesk name also changing, not just client friendly name as per ticket?

Comment (lps88) [reply src/_data/clients.json:14]: Steve asked for a friendly name but Craig removed the charac one and created the_pharmacy_app, the screens hot should be on the ticket really, not sure if vimal added it

Comment (ramuned-NHSD) [reply src/_data/clients.json:14]: Nope, nothing on the ticket. Just says to change friendly name.

**Review by vickicole — APPROVED** [review]

**Review by elokachima1-nhs — APPROVED** [review]

**Pull request merged**